web-server/v0.4/src/utils/parse.js

```diff
@@ -300,3 +316,74 @@ export const generateIterationClusters = (config, iterations) => {
 
   return parseClusteredIterations(primaryMetricIterations, clusterLabels, selectedConfig);
 };
+
+export const getComparisonColumn = (record, iterationNo) => {
+  const children = [];
+  for (let i = 0; i < iterationNo; i += 1) {
```

Comment:
Can we consider using `forEach()` here to maintain the ES6 usage throughout the codebase? For more on `forEach()`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach

Reply:
@gurbirkalsi , the forEach() method executes a provided function for each array element. And here we have to iterate over an integer number. So should we consider using recursion here?

Reply:
Sorry, I thought `iterationNo` was an array here - this looks good!